css/style.css

```diff
@@ -1,4 +1,3 @@
-<<<<<<< HEAD
 main {
```

Comment:
May you kindly add CSS media queries. They allow for responsiveness across a wide range media devices.

Reply:
You did a good job adding a polite review with an explanation as well, please kindly add another inline comment review as suggested by the previous reviewers 😄